k8s/wordpress/README.md

```diff
@@ -8,6 +8,22 @@ WordPress is web publishing software used to create websites and blogs.
 
 Popular open stacks on Kubernetes packaged by Google.
 
+## Design
+
+![Architecture diagram](resources/wordpress-k8s-app-architecture.png)
+
+### Solution Information
+
+WordPress solution exposes its interface on port 80.
+
+Separate StatefulSet Kubernetes objects are used to manage instance of WordPress and instance of MySQL database.
+Single instance of WordPress is deployed as a single POD via Kubernetes StatefulSet.
+
+WordPress instance connects to MySQL database over 3306 port. WordPress stores information in MySQL in `wordpress` database.
+Single instance of MySQL is deployed as a POD via Kubernetes StatefulSet.
+
+WordPress application stores password for MySQL root in the root-password secret and login and password information to access `wordpress` database in wp-user and wp-password secrets (respectively).
```

Comment:
`s/root-password/root_password`

Please [inline code](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting) in order to highlighting the secret key (`root-password`).